ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -565,7 +581,7 @@ index 0000000..24ab0ff
 +    .init           = eb_enc_init,
 +    .encode2        = eb_encode_frame,
 +    .close          = eb_enc_close,
-+    .capabilities   = AV_CODEC_CAP_DELAY | AV_CODEC_CAP_AUTO_THREADS,
```

Comment:
The purpuse of removing AV_CODEC_CAP_AUTO_THREADS is to reset active_thread_type to 0 in validate_thread_parameters() function? Thanks!

Reply:
The original "-threads" doesn't fix ours needs, because it has a limitation of <=16 (MAX_AUTO_THREADS)
.
if (avctx->thread_count > MAX_AUTO_THREADS)
        av_log(avctx, AV_LOG_WARNING,
               "Application has requested %d threads. Using a thread count greater than %d is not recommended.\n",
               avctx->thread_count, MAX_AUTO_THREADS);

Hence, I added the "-thread-count" for our codec wrapper. AV_CODEC_CAP_AUTO_THREADS  is for that "-threads". No need to declare that.